src/js_schema.hpp

```diff
@@ -76,56 +76,59 @@ Property Schema<T>::parse_property(ContextType ctx, ValueType attributes, std::s
     static const String object_type_string = "objectType";
     static const String optional_string = "optional";
     static const String property_string = "property";
-    
+
     Property prop;
     prop.name = property_name;
-    
+
     ObjectType property_object = {};
     std::string type;
-    
+
+    using realm::PropertyType;
+    PropertyType is_optional = PropertyType::Required;
```

Comment:
Maybe `is_optional` is not the best name. Reading "optional is equal to required" was too hard for me :smile: On the other hand, `modifier` is too generic.